distribution/docs/src/main/resources/content/_installing/installing-intro.adoc

```diff
@@ -23,6 +23,11 @@ Adding other components does not have any security/hardening implications.
 
 === Installation Prerequisites
 
+[WARNING]
```

Comment:
❓ ❗️ Will that be inherited by all upstream projects? If so than it would be a problem for those that do not enable the security manager.

Reply:
I think that using the security manager should be considered the norm not the exception.  I will leave the documentation as-is.